cmds/installcommand/installcommand.go

```diff
@@ -135,6 +135,10 @@ func main() {
 	cmd.Stderr = os.Stderr
 	cmd.Stdout = os.Stdout
 	if err := cmd.Run(); err != nil {
-		log.Fatal(err)
+		exitErr, ok := err.(*exec.ExitError)
+		if !ok {
+			log.Fatal(err)
+		}
+		os.Exit(exitErr.Sys().(syscall.WaitStatus).ExitStatus())
```

Comment:
ah, oopsie. This is going to need to go in an _linux.go file. Sys() IIUC is pretty system-dependent. 

Man, what a mess.